### PR TITLE
feat: use java queue for events logging

### LINKED ISSUE #131: Add events on artifact upload
Let's integrate events queue and add events on artifact upload.

### PR DESCRIPTION
**Pull request by olenagerasimova**
closes #131

### CODE REVIEW

File: src/test/java/com/artipie/debian/DebianSliceITCase.java [hunk 1]
```diff
@@ -89,12 +91,12 @@ public final class DebianSliceITCase {
...
      * Artifact events queue.
      */
-    private EventQueue<ArtifactEvent> events;
+    private Queue<ArtifactEvent> events;
 
     @BeforeEach
     void init() throws IOException, InterruptedException {
         this.storage = new InMemoryStorage();
-        this.events = new EventQueue<>();
+        this.events = new LinkedList<>();
```
Comment (dgarus) [inline src/test/java/com/artipie/debian/DebianSliceITCase.java:99]:
```suggestion
        this.events = new ConcurrentLinkedQueue<>();
```

File: src/test/java/com/artipie/debian/DebianSliceITCase.java [hunk 2]
```diff
@@ -23,6 +22,9 @@
 import java.net.URL;
 import java.nio.file.Files;
 import java.nio.file.Path;
+import java.util.LinkedList;
+import java.util.Optional;
```
Comment (dgarus) [inline src/test/java/com/artipie/debian/DebianSliceITCase.java:26]:
```suggestion
import java.util.Optional;
import java.util.concurrent.ConcurrentLinkedQueue;
```

**Review by dgarus — DISMISSED** [review]
Thanks!
Please look at my comments

**Comment by olenagerasimova** [comment]
thanks, corrected

**Comment by olenagerasimova** [comment]
@dgarus please, check

**Review by dgarus — APPROVED** [review]
Thanks! lgtm

**Pull request merged**